sdk/src/main/java/com/meniga/sdk/MenigaSettings.java

```diff
@@ -295,13 +296,13 @@ public Builder persistenceMode(PersistenceMode persistenceMode) {
 		}
 
 		/**
-		 * Sets the OkHttp read and write timeout limit in ms
+		 * Sets the OkHttp read and write timeoutInSeconds limit in ms
```

Comment:
IDE overdid it :) `timeoutInSeconds` -> `timeout`.

Reply:
I specifically asked for that not to happen in one of the pop ups, sigh. Fixed